etc/deprecated_rules.json

```diff
@@ -1 +1,7 @@
-{}
\ No newline at end of file
+{
+  "3a86e085-094c-412d-97ff-2439731e59cb": {
+    "deprecation_date": "2021-03-03",
+    "rule_name": "Setgid Bit Set via chmod",
+    "stack_version": "7.13"
```

Comment:
from the name, it's not clear whether this was the first version where this was removed, or the last version this was included

Reply:
That's a good point. I think I can better handle this when expanding for #958, so I will punt this to that